src/components/login/LoginPage.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {useForm} from 'react-hook-form';
+import Input from '../input';
+import Button from '../button';
+
+function LoginPage() {
+    const {register, handleSubmit, reset, formState: {errors}} = useForm()
+  
+  const submit = (data:any)=> {
+    console.log(data);
+    reset
```

Comment:
- The console.log can be removed
- reset is a function, why is it not being called?